src/scripts/safari/SafariBattle.ts

```diff
@@ -26,6 +26,31 @@ class SafariBattle {
         SafariBattle.text('What will you do?');
         SafariBattle.escapeAttempts = 0;
         $('#safariBattleModal').modal({ backdrop: 'static', keyboard: false });
+
+        // Shiny
+        const location = `${GameConstants.Region[Safari.activeRegion()].charAt(0).toUpperCase() + GameConstants.Region[Safari.activeRegion()].slice(1)} Safari`;
```

Comment:
You can just use `camelCaseToString` here
```suggestion
        const location = `${GameConstants.camelCaseToString(GameConstants.Region[Safari.activeRegion()])} Safari`;
```